client/me/help/help-contact/index.jsx

```diff
@@ -204,6 +215,24 @@ module.exports = React.createClass( {
 
 	onOperatorsAway: function() {
 		this.showOperatorAvailabilityNotice( false );
+
+		//Autofill the subject field since we will be showing it now that operators have went away.
+		this.autofillSubject();
+	},
+
+	/**
+	 * Auto fill the subject with the first five words contained in the message field of the contact form.
+	 */
+	autofillSubject: function() {
+		if ( ! savedContactForm.message || savedContactForm.subject ) {
+			return;
+		}
+
+		const words = savedContactForm.message.split( /\s+/ );
+
+		savedContactForm.subject = words.slice( 0, 5 ).join( ' ' ) + '…';
+
+		this.forceUpdate();
```

Comment:
Probably this happened to prevent the warning, but I don't think we want this behavior here. With `forceUpdate()` we have to be careful because it does just what it says - it _forces_ a re-render regardless of any other circumstance, even if nothing changes. The better way to do this would be to only update if something changes.

:question: what drove the move of `savedContactForm` out of `state`?

```js
if ( ! isEqual( this.state.savedContactForm, savedContactForm ) ) {
	this.setState( { savedContactForm } );
}
```

Now, one might ask why I would recommend doing a deep-compare of an object since that's typically an _expensive_ operation, but we have to remember that a re-render in React involves running the render functions down the tree hierarchy, doing lots of comparisons and evaluation, a virtual DOM diffing, and DOM interaction, so this is probably going to take significantly less time than a React update.